Formula/libetpan.rb

```diff
@@ -0,0 +1,23 @@
+
+class Libetpan < Formula
+  desc "Mail library is to provide a portable, efficient framework."
+  homepage "https://www.etpan.org/libetpan.html"
+  url "https://github.com/dinhviethoa/libetpan/archive/1.7.2.tar.gz"
+  sha256 "32797282a420f3174f4a679548e20fa2bb4acb404b827d62c2f44d3de4eb3120"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "./autogen.sh", "--disable-debug",
+                           "--disable-dependency-tracking",
+                           "--disable-silent-rules",
+                           "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/libetpan-config", "--version"
```

Comment:
Don't retain the test directory, just include a short example (up to 10 lines) in C inside the `test` block of the formula. Look, for example, at how `zita-convolver.rb` does this.